### PR TITLE
Fix pylint warnings

### PR DESCRIPTION
**Pull request by c-w**
PyLint is a great static analyzer for Python - it includes PEP8 compliance checks and a bunch of [more advanced analyses](http://pylint-messages.wikidot.com/all-codes) that improve code quality and reduce the potential for bugs.

There's really no good reason why any project shouldn't be green under PyLint!

### CODE REVIEW

**Comment by VikParuchuri** [comment]
Thanks!  That makes sense, and good idea.  This patch appears to have a couple of pep8 issues, though -- see http://unite.dataquest.io/f/status/62c58e651144a89a35435606a7ef155ea404697a/ for details.

File: split_data.py [hunk 1]
```diff
@@ -10,34 +11,38 @@
...
     # Split the data into reviews.
-    reviews = data.split("\n\n")
+    REVIEWS = DATA.split("\n\n")
     # Randomly shuffle the reviews.
-    random.shuffle(reviews)
+    random.shuffle(REVIEWS)
 
     # Downsample the data to run faster
-    reviews = reviews[:40000]
+    REVIEWS = REVIEWS[:40000]
```
Comment (VikParuchuri) [inline split_data.py]: Why upper case for these?  They aren't constants, and I thought upper case was reserved for those.

**Comment by VikParuchuri** [comment]
We might also want to add a pylint check into test.py so violations are flagged by CI.  What do you think?

File: split_data.py (hunk 1 above)
Comment (c-w) [inline split_data.py]: Curious. Must be a false positive from pylint.

**Comment by c-w** [comment]
Curious about the pep8 issues - when I run the pep8 tool locally, I don't get any violations. What version of pep8 are you using on your continuous integration server?

**Comment by c-w** [comment]
About pylint/pep8 - what do you think about running these checks as a pre-commit hook instead of on the continuous integration server? This would mean that no non-compliant code ever gets close to the revision history.

**Pull request closed without merging**